src/runtime_src/hip/api/hip_module.cpp

```diff
@@ -34,7 +34,7 @@ hip_module_launch_kernel(hipFunction_t f, uint32_t /*gridDimX*/, uint32_t /*grid
   auto cmd_hdl = insert_in_map(command_cache,
                                std::make_shared<kernel_start>(hip_stream,
                                                               hip_func,
-                                                              kernelParams));
+                                                              kernelParams, extra));
```

Comment:
Are we always going to create kernel_start command with 4 arguments? i dont see how can we have kernel_start call with only three arguments without "extra"